src/models/orders.ts

```diff
@@ -16,30 +19,35 @@ export interface Order {
   /**
    * Order can't be matched after this block timestamp (optional)
    **/
-  endBlockTimestamp?: number
-  type: 'RARIBLE_V1' | 'RARIBLE_V2'
-  data?: SellData
-  fill?: number
-  makeStock?: number
-  cancelled?: boolean
-  createdAt?: string
-  lastUpdateAt?: string
-  signature?: string
-  hash?: string
+  endBlockTimestamp?: number;
+
+  data?: OrderData | string;
```

Comment:
This data attribute needs to be a hashed when canceling an order. that's why is can also be a string.